dynamodump.py

```diff
@@ -388,7 +396,8 @@ def do_restore(conn, sleep_interval, source_table, destination_table, write_capa
 
   if dest_table.find("*") != -1:
     matching_destination_tables = get_table_name_matches(conn, dest_table, prefix_separator)
-    logging.info("Found " + str(len(matching_destination_tables)) + " table(s) in DynamoDB host to be deleted: " + ", ".join(matching_destination_tables))
+    deleteStr = ": " if args.dataOnly else " to be deleted: "
```

Comment:
Minor one, but could you please change this to `delete_str` to match the convention of the rest of the file?